### ISSUE TITLE
Help - 3 phase

### ISSUE THREAD

**Issue by rmoosa** [denvera/esp32-nanoview#2]
Hi. Sorry for logging an issue. Wasn't sure how else to reach out. 
Just wanted to see if u have any thoughts on this. 
I need to measure loads on a 3 phase db board. 
I noted a comment on the Nanoview faq page regarding 3 phase. 
https://www.nanoview.co.za/faq.html

I initially thought that if I wire up the hub to one phase and hook up cts circuits on the other phases, it would measure the amps and do the conversion based on voltage on the phase it is connected to. 
So if voltages are the same on all 3 phases, I would have accurate readings. And if voltages differed by about 2 or 3 %. Then the readings would be out by that amount. 
That would have been good enough for me. This would be similar to the efergy. In the efergy, u set the voltage on the app, and it uses that set figure for the calcs. 

But it appears that my assumptions are wrong. Seems like it may not just be reading voltage, I'm guessing it is also reading the phase cycles and since the phases are offset by 120 degrees, the calc are totally out. 

Long story short. The Nanoview faq suggests putting separate devices on each phase. I am planning to do that. But I wanted to see if I could get away with wiring them all up to a single esp32 and create 3 separate serial channels to read the 3 devices. 
Any thought, potential pitfalls or hints you can offer on this approach. 
Or would it be simpler to just use 3 esp devices and feed all of the output to a common Mqtt broker.

**Comment by rmoosa** [comment]
Hi. Dunno if u saw the last message. 
Do u have any thoughts on this.

**Comment by denvera** [comment]
Hi @rmoosa 

Thanks for your interest in the project. I suspect its possible to use a single ESP32, given the ESP32 appears to have 3 UART controller based on the [documentation ](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/uart.html#:~:text=The%20ESP32%20chip%20has%20three%20UART%20controllers).

If you're keen to test I can try updating the project to support this.

**Comment by rmoosa** [comment]
Yes I am very keen to test. 
I already have 3 Nanoview s on hand and will be installing them soon. Probably over the weekend.

**Comment by rmoosa** [comment]
Hi. Just following up to see if u have had a chance to look into this. 
I have already mounted all 3 u it's into the db and I am waiting to do the testing. 
Thanks

**Comment by rmoosa** [comment]
Hi. 
Have you managed to take a look at this yet?
Thank
Ridwan
